### PR TITLE
Update pmcli.py

### PR DESCRIPTION
**Pull request by SteveKueng**
possibility to add the placeholder to a second group

### CODE REVIEW

**Comment by MagerValp** [comment]
Good idea, but there are some issues with the commit, check the code comments.

**Comment by SteveKueng** [comment]
now it should work with multiple groups :)

File: pmcli.py [hunk 1]
```diff
@@ -91,14 +91,14 @@ def do_add_placeholder(pm, args):
     if id_type not in ("serial", "imei", "meid", "udid"):
         sys.exit(usage)
     try:
-        group = args[2]
+    	group = args[2]
     except IndexError:
         group = None
```
Comment (MagerValp) [inline pmcli.py]: Lines 93-96 and the if clause on line 98 are redundant, since args[2:] will return an empty list if there are no groups to add.

**Comment by MagerValp** [comment]
Looking good, removing 93-96 and 98 should make it ready to merge.

PS: looks like a hard tab snuck in on line 94, make sure you check for that before committing.

**Comment by SteveKueng** [comment]
so, I changed this

sorry that it took so long... problems with the git client...

**Comment by MagerValp** [comment]
Looking good, thank you, merging.

**Pull request merged**